src/components/billboard.js

```diff
@@ -0,0 +1,66 @@
+const isMobileVR = AFRAME.utils.device.isMobileVR();
+
+// Billboard component that only updates visible objects and only those in the camera view on mobile VR.
```

Comment:
Why did you enable this optimization only on mobile?

Reply:
Basically I've been adding optimizations that technically degrade the experience (eg this one causes weird billboarding behavior if you move your camera quickly, since there's latency before it is corrected) so I think we should only apply these to mobile VR contexts since that is where we have the main sensitivity and susceptibility to lower frame rates

Reply:
I slightly prefer consistency between platforms, but it probably doesn't matter much.